### PR TITLE
FAQ - update 'international_phone_numbers' entry'

### PR DESCRIPTION
**Pull request by heinezen**
Changelog:
- Include information from who_can_help
- Add national hotline numbers
- Add description about cost of calling

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -31,17 +31,21 @@
...
-                            "Technical hotline: +49 30 498 75401 available Monday to Saturday from 7 a.m. to 10 p.m. German local time (except on national holidays)",
-                            "TAN hotline: +49 30 498 75402 available 24 hours a day, seven days a week",
-                            "Note: These numbers are <b>not free of charge</b> - the cost of the international call is based on the contract with your provider."
+                            "You can reach us by phone from within Germany and from abroad. The national numbers are free of charge, but are only available from within Germany. The international numbers are <b>not free of charge</b> - the cost of the international call is based on the contract with your provider.",
+                            "<b>Technical Hotline</b>",
+                            "The technical hotline will help you with technical questions about the Corona-Warn-App, for instance if you have problems with risk identification. It is available Monday to Saturday from 7 a.m. to 10 p.m. German local time (except on national holidays)",
```
Comment (Ein-Tim) [inline src/data/faq.json:40]: "risk identification" -> "exposure logging"

**Review by Ein-Tim — CHANGES_REQUESTED** [review]

File: src/data/faq_de.json [hunk 2]
```diff
@@ -31,14 +31,19 @@
...
+                        "title": "So erreichen Sie uns - Internationale Telefonnummern",
                         "anchor": "international_phone_numbers",
                         "active": true,
                         "textblock": [
-                            "Außerhalb Deutschlands erreichen Sie uns unter folgenden internationalen Telefonnummern:",
-                            "Technische Hotline: +49 30 498 75401 erreichbar Montag bis Samstag von 7 bis 22 Uhr Ortszeit in Deutschland (außer an bundesweiten Feiertagen)",
-                            "TAN-Hotline: +49 30 498 75402 erreichbar 24 Stunden und sieben Tage die Woche",
-                            "Hinweis: Diese Rufnummern sind <b>nicht kostenfrei</b> - die Kosten für den internationalen Anruf richten sich nach dem Vertrag mit Ihrem Provider."
+                            "Sie können uns per Telefon von in und außerhalb Deutschlands erreichen. Die nationalen Telefonnummern sind für Sie kostenfrei, können jedoch nur aus dem deutschen Telefonnetz erreicht werden. Die internationalen Rufnummern sind <b>nicht kostenfrei</b> - die Kosten für den internationalen Anruf richten sich nach dem Vertrag mit Ihrem Provider.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:38]: Maybe 
"von in und außerhalb" =>
"von inner- und außerhalb" ?

**Review by MikeMcC399 — COMMENTED** [review]
Just one suggestion for the German text.

File: src/data/faq.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq.json]: fixed

File: src/data/faq_de.json (hunk 2 above)
Comment (heinezen) [reply src/data/faq_de.json]: yes good idea :)

**Review by Ein-Tim — APPROVED** [review]
LGTM now!

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
image
add missing </b>

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 @heinezen Thanks!

**Pull request merged**